stackit/internal/services/ske/cluster/resource.go

```diff
@@ -552,7 +552,21 @@ func (r *clusterResource) Create(ctx context.Context, req resource.CreateRequest
 	ctx = tflog.SetField(ctx, "project_id", projectId)
 	ctx = tflog.SetField(ctx, "name", clusterName)
 
-	availableVersions, err := r.loadAvaiableVersions(ctx)
+	// If SKE functionality is not enabled, enable it
+	_, err := r.client.EnableService(ctx, projectId).Execute()
+	if err != nil {
+		core.LogAndAddError(ctx, &resp.Diagnostics, "Error creating project", fmt.Sprintf("Calling API: %v", err))
+		return
+	}
+
+	model.Id = types.StringValue(projectId)
```

Comment:
This is not correct. `model.Id` has the format `project_id, cluster_name` and is set inside the `mapFields` method

Reply:
So you can remove this line